apps/src/code-studio/assets/show.js

```diff
@@ -5,8 +5,6 @@ var React = require('react');
 var ReactDOM = require('react-dom');
 var ImagePicker = require('../components/ImagePicker');
 var SoundPicker = require('../components/SoundPicker');
-var LibraryPicker = require('../components/LibraryPicker');
```

Comment:
This line is being removed by this PR: https://github.com/code-dot-org/code-dot-org/pull/29381/files
(same with line 50)